### PR TITLE
Increase the timeout to address the flakiness

### PR DESCRIPTION
**Pull request by haiyanmeng**

### CODE REVIEW

File: e2e/testcases/status_enablement_test.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func TestStatusEnabledAndDisabled(t *testing.T) {
 			resourceGroupHasNoStatus,
 			testpredicates.HasLabel(common.InventoryLabel, id),
 		},
-		testwatcher.WatchTimeout(120*time.Second))
+		testwatcher.WatchTimeout(180*time.Second))
```
Comment (sdowell) [inline e2e/testcases/status_enablement_test.go:66]: Is there any reason to not use `DefaultWaitTimeout`?

Comment (haiyanmeng) [reply e2e/testcases/status_enablement_test.go]: Switched to `DefaultWaitTimeout`. Thanks for the suggestion.

**Review by janetkuo — APPROVED** [review]
/lgtm

**Pull request merged**